### PR TITLE
Update Version To 2024.1.0.0 and add TF frontend

### PR DESCRIPTION
**Pull request by mryzhov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_.yaml [hunk 1]
```diff
@@ -14,8 +14,12 @@ cxx_compiler_version:
 - '12'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+libabseil:
+- '20240116'
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```
Comment (ilya-lavrenov) [inline .ci_support/linux_64_.yaml]:
```suggestion
- 2024.1.0
```

File: .ci_support/linux_aarch64_.yaml [hunk 2]
```diff
@@ -18,8 +18,12 @@ cxx_compiler_version:
 - '12'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+libabseil:
+- '20240116'
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```
Comment (ilya-lavrenov) [inline .ci_support/linux_aarch64_.yaml]:
```suggestion
- 2024.1.0
```

File: .ci_support/osx_64_.yaml [hunk 3]
```diff
@@ -12,8 +12,12 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '16'
+libabseil:
+- '20240116'
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```
Comment (ilya-lavrenov) [inline .ci_support/osx_64_.yaml]:
```suggestion
- 2024.1.0
```

File: .ci_support/osx_arm64_.yaml [hunk 4]
```diff
@@ -12,8 +12,12 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '16'
+libabseil:
+- '20240116'
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```
Comment (ilya-lavrenov) [inline .ci_support/osx_arm64_.yaml]:
```suggestion
- 2024.1.0
```

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

File: .ci_support/win_64_.yaml [hunk 5]
```diff
@@ -6,8 +6,12 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240116'
 libopenvino_dev:
-- 2023.3.0
+- 2024.0.0
```
Comment (ilya-lavrenov) [inline .ci_support/win_64_.yaml]:
```suggestion
- 2024.1.0
```

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
Replaced by:
- https://github.com/conda-forge/openvino-tokenizers-feedstock/pull/12
- https://github.com/conda-forge/openvino-tokenizers-feedstock/pull/13

**Pull request closed without merging**